src/pages/Models/Details/_model-details.scss

```diff
@@ -226,6 +230,7 @@
   .entity-icon {
     border-radius: 50%;
     height: 1.5rem;
+    margin-bottom: 0.15rem;
```

Comment:
The icons are a bit too high in the row.

![Screen Shot 2021-01-12 at 11 40 07 AM](https://user-images.githubusercontent.com/532033/104352331-2a9e2180-54cc-11eb-905d-120209f8a9df.png)